### PR TITLE
support materialized views in longevity test

### PR DESCRIPTION
**Pull request by larisau**

### CODE REVIEW

**Comment by roydahan** [comment]
Looks good.
However, let's have a seperate branch with all these longevity enhancements, so we can test them all together.

File: tests/gce-longevity-1.7-1TB-7days.yaml [hunk 1]
```diff
@@ -35,7 +37,22 @@ backends: !mux
         scylla_repo: 'https://s3.amazonaws.com/downloads.scylladb.com/rpm/unstable/centos/branch-1.7/17/scylla.repo'
         us_east_1:
           gce_datacenter: 'us-east1-b'
-
+    aws: !mux
```
Comment (amoskong) [inline tests/gce-longevity-1.7-1TB-7days.yaml]: We use different for AWS and GCE, and the filename prefix is `gce-`.

File: longevity_test.py [hunk 2]
```diff
@@ -37,13 +38,18 @@ def test_custom_time(self):
...
                                     monitoring_set=self.monitors)
         stress_queue = list()
-        for cmd in ('stress_cmd', 'stress_cmd_1'):
-            stress_cmd = self.params.get(cmd)
-            if stress_cmd:
-                if 'counter_' in stress_cmd:
-                    self._create_counter_table()
-                self.log.debug('stress cmd: {}'.format(stress_cmd))
-                stress_queue.append(self.run_stress_thread(stress_cmd=stress_cmd))
+        for stress_cmd in self.params.get('stress_cmd'):
```
Comment (amoskong) [inline longevity_test.py]: It doesn't support original type (single str).
Let's check if it's a list, then process two kinds of case.

**Comment by larisau** [comment]
Merged to https://github.com/scylladb/scylla-cluster-tests/tree/longevity

**Comment by amoskong** [comment]
The longevity branch had been merged to master. So let's close the PR.

**Pull request closed without merging**